Integrations/integration-Wildfire.yml

```diff
@@ -240,45 +292,58 @@ script:
-          var res = http(
-              testUrl,
-              {
-                  Method: 'POST',
-                  Body: 'apikey='+TOKEN+'&format=xml&hash=7f638f13d0797ef9b1a393808dc93b94',
-                  Headers: DEFAULT_HEADERS
-              },
-              params.insecure,
-              params.proxy
-          );
+        case 'test-module': // This is the call made when pressing the integration test button.
+            testUrl = URL + URL_DICT.report;
+            var res = http(
+                testUrl,
+                {
+                    Method: 'POST',
+                    Body: 'apikey='+TOKEN+'&format=xml&hash=7f638f13d0797ef9b1a393808dc93b94',
+                    Headers: DEFAULT_HEADERS
+                },
+                params.insecure,
+                params.proxy
+                );
+            if(res.StatusCode===200){
+                return 'ok';
+            }
+            else{
+                return 'Something went wrong.\n Status code: ' + res.StatusCode + '\nBody: '+ JSON.stringify(res) + '.';
+            }
+            break;
 
-          if(res.StatusCode===200)
-              return 'ok';
-          else
-              return 'Something went wrong.\n Status code: ' + res.StatusCode + '\nBody: '+ JSON.stringify(res) + '.';
-          break;
-      case 'file':
-      case 'wildfire-report':
-        var reportResponse = getReport(args.md5, args.hash, args.file);
-        if (reportResponse.error){
-            return reportResponse.error;
-        }
-        return createReport(reportResponse, args.format, args.verbose);
-      case 'wildfire-upload':
-          var response = uploadFile(args.upload);
-          var uploadData = response.data;
-          var result = response.res;
-          return createEntry(uploadData,'WildFire Report', result);
-      case 'detonate-file':
-          return detonate('file', args.upload, args.format, args.delay, args.timeout, args.verbose);
-      case 'detonate-file-remote':
-          return detonate('remoteFile', args.upload, args.format, args.delay, args.timeout);
+        case 'wildfire-upload':
+            var response = uploadFile(args.upload);
+            var uploadData = response.data;
+            var result = response.res;
+            return createUploadEntry(uploadData,'WildFire Upload File', result);
+
+        case 'wildfire-upload-file-remote':
+            var response = uploadFileRemote(args.upload);
+            var uploadData = response.data;
+            var result = response.res;
+            return createUploadEntry(uploadData,'WildFire Upload File Remote', result);
+
+        case 'file':
+        case 'wildfire-report':
+            inputs = GetReportArgsHandler(args.md5, args.hash, args.file)
+            entries = []
+            for (i=0; i < inputs.length; i++) {
+                var reportResponse = getReport(inputs[i]);
+                if (reportResponse.error){
+                    return reportResponse.error;
```

Comment:
you should also return the successful results so far (or consider to continue to the rest of the inputs instead of stopping at failures)